extensions/applicationinsights-debugplugin-js/src/InternPlugin.ts

```diff
@@ -0,0 +1,252 @@
+/**
+ * DebugPlugin.ts
+ * @copyright Microsoft 2020
+ */
+
+import {
+  BaseTelemetryPlugin, IConfiguration, CoreUtils,
+  IAppInsightsCore, IPlugin, ITelemetryItem, IProcessTelemetryContext, _InternalLogMessage, LoggingSeverity, _InternalMessageId, getNavigator,
+  ITelemetryPluginChain, InstrumentFunc, IInstrumentHooksCallbacks, IInstrumentCallDetails, InstrumentFuncs
+} from '@microsoft/applicationinsights-core-js';
+import { LoggerElement } from './components/helpers';
+import { tempStyle, permStyle } from './components/styleNodeSrc';
+import { DebugBin, DebugBinParent } from './components/debugBins';
+
+
+export default class DebugPlugin extends BaseTelemetryPlugin {
+  /**
+   * the root element of the logger
+   */
+  public myRootElement: HTMLDivElement;
+
+  /**
+   * the logging element
+   */
+  public myLogger: LoggerElement;
+
+  /**
+   * the style that is only on the webpage when the log is active
```

Comment:
Rename to loggingElement maybe?, this could cause confusion with DiagnosticLogger property of the base class

Reply:
good suggestion - fixed